src/main/java/com/conveyal/datatools/manager/models/FeedSource.java

```diff
@@ -538,4 +544,88 @@ public FeedSource clone () throws CloneNotSupportedException {
         return (FeedSource) super.clone();
     }
 
+    @Override
```

Comment:
If we just stash the feed ID in the multimap, we wouldn't need this equality method.

Reply:
True